pandas/tests/io/test_parquet.py

```diff
@@ -252,6 +301,10 @@ def test_duplicate_columns(self, fp):
 
     def test_unsupported(self, fp):
 
+        # bool with none
+        df = pd.DataFrame({'a': [True, None, False]})
```

Comment:
It is not expected to https://github.com/dask/fastparquet/issues/105
Since situations like this are typically cast to float in pandas, it didn't seem reasonable to add additional code-paths to int-with-None and bool-with-None columns.